packages/prosess-vedtak/src/components/VedtakAvslagArsakOgBegrunnelsePanel.jsx

```diff
@@ -1,110 +0,0 @@
-import { TextAreaField } from '@fpsak-frontend/form';
-import {
-  decodeHtmlEntity,
-  getLanguageCodeFromSprakkode,
-  hasValidText,
-  maxLength,
-  minLength,
-  requiredIfNotPristine,
-  getKodeverknavnFn,
-} from '@fpsak-frontend/utils';
-import { VerticalSpacer } from '@fpsak-frontend/shared-components';
-
-import { hasIkkeOppfyltSoknadsfristvilkar } from './VedtakHelper';
-
-import styles from './vedtakAvslagPanel.less';
-import vedtakVarselPropType from '../propTypes/vedtakVarselPropType';
-
-const maxLength1500 = maxLength(1500);
-const minLength3 = minLength(3);
-
-export const getAvslagArsak = (vilkar, aksjonspunkter, vedtakVarsel, getKodeverknavn) => {
-  const avslatteVilkar = vilkar.filter(
-    v =>
-      Array.isArray(v.perioder) &&
-      v.perioder.some(periode => periode.vilkarStatus.kode === vilkarUtfallType.IKKE_OPPFYLT),
-  );
-  if (avslatteVilkar.length === 0) {
-    return <FormattedMessage id="VedtakForm.UttaksperioderIkkeGyldig" />;
-  }
-
-  return `${getKodeverknavn(avslatteVilkar[0].vilkarType)}: ${getKodeverknavn(
-    vedtakVarsel.avslagsarsak,
-    avslatteVilkar[0].vilkarType.kode,
-  )}`;
-};
-
-export const VedtakAvslagArsakOgBegrunnelsePanel = ({
-  intl,
-  behandlingStatusKode,
-  vilkar,
-  aksjonspunkter,
-  sprakkode,
-  readOnly,
-  alleKodeverk,
-  vedtakVarsel,
-}) => {
-  const getKodeverknavn = getKodeverknavnFn(alleKodeverk, kodeverkTyper);
-  return (
-    <>
-      {getAvslagArsak(vilkar, aksjonspunkter, vedtakVarsel, getKodeverknavn) && (
-        <div>
-          <Undertekst>{intl.formatMessage({ id: 'VedtakForm.ArsakTilAvslag' })}</Undertekst>
-          <Normaltekst>{getAvslagArsak(vilkar, aksjonspunkter, vedtakVarsel, getKodeverknavn)}</Normaltekst>
-          <VerticalSpacer sixteenPx />
-        </div>
-      )}
-      {behandlingStatusKode === behandlingStatus.BEHANDLING_UTREDES && hasIkkeOppfyltSoknadsfristvilkar(vilkar) && (
-        <Row>
-          <VerticalSpacer sixteenPx />
-          <Column xs="7">
-            <TextAreaField
-              name="begrunnelse"
-              label={intl.formatMessage({ id: 'VedtakForm.Fritekst' })}
-              validate={[requiredIfNotPristine, minLength3, maxLength1500, hasValidText]}
-              maxLength={1500}
-              readOnly={readOnly}
-              badges={[
-                {
-                  type: 'fokus',
-                  textId: getLanguageCodeFromSprakkode(sprakkode),
-                  title: 'Malform.Beskrivelse',
-                },
-              ]}
-            />
-          </Column>
-        </Row>
-      )}
-      {readOnly && vedtakVarsel.avslagsarsakFritekst != null && (
-        <span>
-          <VerticalSpacer twentyPx />
-          <Undertekst>{intl.formatMessage({ id: 'VedtakForm.Fritekst' })}</Undertekst>
-          <VerticalSpacer eightPx />
-          <div className={styles.fritekstItem}>{decodeHtmlEntity(vedtakVarsel.avslagsarsakFritekst)}</div>
-        </span>
```

Comment:
@baskevold @hallvardastark  jeg har erstattet dette med `<AvslagÅrsakListe>` som kun viser avslageneårsakene (slik som for førstegangvurdering/VedtakForm). Forskjellen er at vi da ikke viser  `begrunnelse` og `avslagsarsakFritekst`. Men det tror jeg ikke er i bruk, hvis ikke det er noe greier ifm Informasjonsbehov da....